src/v2/auth.rs

```diff
@@ -365,7 +343,7 @@ mod tests {
     fn basic_realm_parses_correctly() -> Result<()> {
         let realm = "Registry realm";
 
-        let header_value = HeaderValue::from_str(&format!(r#"Basic realm="{}""#, realm))?;
+        let header_value = HeaderValue::from_str(&format!(r#"Basic realm="{}""#, realm)).unwrap();
```

Comment:
My rule of thumb is that if it's a 3rd party library I don't rely on it and avoid panicing.

Reply:
Bump.